src/components/manage/Widgets/SelectAutoComplete.jsx

```diff
@@ -139,7 +139,7 @@ class SelectAutoComplete extends Component {
     if (
       this.state.termsPairsCache.length === 0 &&
       value?.length > 0 &&
-      choices.length > 0
+      choices?.length > 0
```

Comment:
ok so if we want to allow null then please remove the default value  = [] as it's not necessary any more.

Also adding a test for the choices=null case would be nice... but might not be trivial, provided that currently there is only a single render test case.

Reply:
done